work_items/vc_marketplace.md

```diff
@@ -0,0 +1,103 @@
+# VC Marketplace Work Item
+
+## Owners
+
+- Kishore Bhatia, Affinidi ([kishore@affinidi.com](mailto:kishore@affinidi.com), @kishorebhatia)
+- Wayne Chang, Spruce Systems
+- Rouven Heck, Consensys Mesh
+- Martin Riedel, Consensys Mesh (martin.riedel@mesh.xyz)
+- Sandeep Bajjuri ([sandeep.b@affinidi.com](mailto:sandeep.b@affinidi.com), @sandeepbajjuri)
+- Stepan Gershuni (@gerstep, stepan.g@affinidi.com)
+
+## Outcome
+
+By establishing the reference architecture for a Verifiable Credentials Marketplace, we would be able to discover, save, share and verify verifiable credentials in an anonymized manner across networks. This enables us to increase the usability and value of VCs across networks
+
+Consider the example of a migrant worker who doesn’t have the possibility to avail a loan due to missing Identification or credit score. With his income credentials generated from a particular employer he will be able to secure a loan from a different provider,
+
+(Mechanics of a real world credential exchange (protocol) covering value exchange can be defined to establish the reference architecture for a Verifiable Credentials Marketplace.)
+
+## Deliverables
+
+High level architecture for Verifiable Credentials Marketplace independent from solution architecture / protocol design that covers the following items:
+
+- Use cases of VC Marketplace
+- VC metadata required to enable the Marketplace
+- Discovery mechanisms:
+    - Who are the issuers of credentials that I need?
+    - What credentials do I need to receive service?
+- Aggregated and anonymous tracking of VC Marketplace activity
+- Privacy, security, and compliance considerations across marketplace actors
```

Comment:
Added this item, and if everyone is okay with the addition, then I'm fine to move this into the WG.